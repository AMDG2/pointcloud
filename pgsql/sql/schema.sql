-- See https://github.com/pgpointcloud/pointcloud/issues/71
select '#71', PC_SchemaIsValid('<xml/>'::xml::text);
