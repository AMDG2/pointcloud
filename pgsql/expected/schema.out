-- See https://github.com/pgpointcloud/pointcloud/issues/71
select '#71', PC_SchemaIsValid('<xml/>'::xml::text);
WARNING:  unable to evaluate xpath expression "/pc:PointCloudSchema/pc:dimension" against schema XML
 ?column? | pc_schemaisvalid 
----------+------------------
 #71      | f
(1 row)

